test/src/utils/apm/assertCanPublish.test.ts

```diff
@@ -0,0 +1,51 @@
+import { assert } from 'chai'
+import { assertCanPublish } from '~/src/utils/apm'
+import { getMainnetProvider } from '~/test/test-helpers/providers'
+import { zeroAddress } from '~/src/params'
+import { BuidlerPluginError } from '@nomiclabs/buidler/plugins'
+
+describe('apm > assertCanPublish', () => {
+  const provider = getMainnetProvider()
```

Comment:
Just wondering, this is fine for now, but do we have other tests where we use a local ganache instance instead?

It would be nice to create an environment specifically for these tests, so we don't have to go to network (especially one that may change on us, if I ever decide to update the finance repo's manager).